### PR TITLE
Minor tweaks for demo

### PR DESCRIPTION
**Pull request by matthampton-stock2shop**
- Navigate to the dashboard from the home URL
- Stop the customers -> orders screen from being full width
- Make the b2b url configurable
- Update the app title and favicon

### CODE REVIEW

File: src/layouts/Console.svelte [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
   $: isSettings = $location.includes('/settings');
   $: isFullWidth =
-    $location.includes('/dashboard') || $location.endsWith('/products') || $location.endsWith('/orders') || $location.endsWith('/customers');
+    $location.startsWith('/dashboard') || $location.startsWith('/products') || $location.startsWith('/orders') || $location.startsWith('/customers');
```
Comment (justinengle-krs) [inline src/layouts/Console.svelte:11]: why this change? Just curious

I was a bit worried about potential routes would be /products/something
But I guess its not going to be a thing

Comment (matthampton-stock2shop) [reply src/layouts/Console.svelte:11]: Then reason was the inverse of that worry (that is actually a thing): `http://localhost:3003/#/customer/6263834/orders` was going full width cos it endsWith('/orders')

**Review by justinengle-krs — APPROVED** [review]
